.github/workflows/prcheck.yml

```
@@ -0,0 +1,104 @@
+name: PR Check
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches: [ main, beta ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)